app/src/hooks/featureLimiter/useFeatureLimiter.ts

```diff
@@ -46,6 +46,8 @@ export const useFeatureLimiter = () => {
   };
 
   const getFeatureLimitValue = (featureLimitType: FeatureLimitType) => {
+    if (!(featureLimitType in FeatureLimitType)) return true; // free feature
+
     return (
       featureLimits[userPlan]?.[featureLimitType] ??
       featureLimits[APP_CONSTANTS.PRICING.PLAN_NAMES.BASIC]?.[featureLimitType] // if plan is not found, return basic plan limit eg: for lite plan
```

Comment:
Did we check for professional plan?? I think this will return false for professional and other premium plans which are not present in the map